src/pages/billing/models/BillingProject.ts

```diff
@@ -18,7 +18,7 @@ export interface BillingProject {
 
 export interface AzureBillingProject extends BillingProject {
   cloudPlatform: 'AZURE'
-  azureManagedAppCoordinates: AzureManagedAppCoordinates
+  managedAppCoordinates: AzureManagedAppCoordinates
```

Comment:
I noticed that this was incorrect, though nothing in TypeScript is referencing it so it wasn't causing any issues.

Reply:
...yet (although now, it just won't)